libs/core/src/lib/select/option/option.component.ts

```diff
@@ -51,35 +50,63 @@ export class OptionComponent implements OnInit {
 
     /** Emitted when the selected state changes. */
     @Output()
-    readonly selectedChange: EventEmitter<OptionComponent>
-        = new EventEmitter<OptionComponent>();
+    readonly selectedChange: EventEmitter<OptionComponent> = new EventEmitter<OptionComponent>();
```

Comment:
Who is going to use this ?  This binding does not have much value  when not connected with the parent `select`

Reply:
Yes, sorry this is a leftover from previous implementation. I left it for eventual backwards compatibility, but since this PR brings breaking changes anyway I'm going to remove it.